modules/default_location/auth/infield_checklist_admin_role.group.yaml

```diff
@@ -0,0 +1,89 @@
+# This role is identical to the normal users, but is used to group user with additional InField feature capabilities for checklist admins.
+# Typical Team captain / Supervisor users
+name: 'infield_{{location_name}}_{{checklist_admin_role_name}}'
+source_id: '{{infield_location_checklist_admin_users}}'
+metadata:
+  origin: 'cdf-project-templates'
+  template_version: '{{template_version}}'
+capabilities:
+  - threedAcl:
+      actions:
+        - READ
+      scope:
+        datasetScope: {
+          ids: [{{location_source_data_dataset}}]
+        }
+  - assetsAcl:
+      actions:
+        - READ
+      scope:
+        datasetScope: {
+          ids: [{{location_source_data_dataset}}]
+        }
+  - filesAcl:
+      actions:
+        - READ
+      scope:
+        datasetScope: {
+          ids: [{{location_source_data_dataset}}, {{location_app_data_dataset}}]
+        }
+  - filesAcl:
+      actions:
+        - WRITE
+      scope:
+        datasetScope: {
+          ids: [{{location_app_data_dataset}}]
+        }
+  - relationshipsAcl:
+      actions:
+        - READ
+      scope:
+        datasetScope: {
+          ids: [{{location_source_data_dataset}}]
+        }
+  - timeSeriesAcl:
+      actions:
+        - READ
+      scope:
+        datasetScope: {
+          ids: [{{location_source_data_dataset}}, {{location_app_data_dataset}}]
+        }
+  - timeSeriesAcl:
+      actions:
+        - WRITE
+      scope:
+        datasetScope: {
+          ids: [{{location_app_data_dataset}}]
+        }
+  - dataModelsAcl:
+      actions:
+        - READ
+      scope:
+        spaceScope: [
+          'APM_Config',
+          'cdf_apm',
+          'cdf_infield',
+          {{apm_datamodel_space}} # from modules/config.yaml
+        ]
+  - dataModelInstancesAcl:
+      actions:
+        - READ
+      scope:
+        spaceScope: [
+          'APM_Config',
+          'cdf_apm',
```

Comment:
Side note: Discussed with FDM team, we won't need `instance read` on the system data models if we remove the `globalNode`s (referenceTemplateItems etc.), which are optional.